examples/src/main/proto/helloworld.proto

```diff
@@ -25,6 +25,7 @@ package helloworld;
 service Greeter {
   // Sends a greeting
   rpc SayHello (HelloRequest) returns (HelloReply) {}
+  rpc SayHelloStreaming (stream HelloRequest) returns (stream HelloReply) {}
```

Comment:
This file is shared by other languages, so we can't "just" change our version. But also, this is used by the helloworld, which is supposed to be super-simple. Let's instead create a new .proto for this streaming example. If you wanted to name it "helloworld_streaming.proto" or similar, that'd be no big deal to me.